lib/templates.rake

```diff
@@ -103,6 +107,23 @@ def update_snippet
   { :diff => diff, :status => status, :result => result }
 end
 
+def get_default_branch
+  # Always use develop on Foreman-nightly
+  return "-b develop" if Foreman::Version.new.tag == "develop"
+
+  url = URI.parse("https://api.github.com/repos/theforeman/community-templates/branches")
```

Comment:
There are ruby/git bindings, by the way.  We use them in https://github.com/theforeman/theforeman.org/blob/gh-pages/scripts/committers.rb, which I notice, er, that you wrote :smile: